spec/system/casework/viewing_an_application/that_is_open_and_unassigned_spec.rb

```diff
@@ -345,4 +345,14 @@
       expect(page).to have_content('Home address Does not have a home address')
     end
   end
+
+  context 'with supporting evidence' do
+    it 'shows a table with supporting evidence' do
+      evidence_row = find(:xpath,
+                          "//table[@class='govuk-table app-dashboard-table govuk-!-margin-bottom-9']
+                          //tr[contains(td[1], 'test.pdf')]")
+
+      expect(evidence_row).to have_content('test.pdf Download file (pdf, 12 Bytes)')
+    end
+  end
```

Comment:
Does there need to be any negative tests e.g. if the supporting evidence file is no longer present for a revision? Or is that better for a future ticket where multiple submissions with different evidence can be tested out?

Reply:
I've added a test to check that if there is no supporting evidence then no supporting evidence section is shown